cds/modules/records/mappings/categories/category-v1.0.0.json

```diff
@@ -9,6 +9,19 @@
         "name": {
           "type": "string"
         },
+        "_access": {
+          "type": "object",
+          "properties": {
+            "read": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "update": {
+              "type": "string",
```

Comment:
hmm this is an array, no?

Reply:
and ``read`` I think

Reply:
Apparently in ES there is not specific `array` data type:

Field types: https://www.elastic.co/guide/en/elasticsearch/reference/2.4/mapping.html

https://www.elastic.co/guide/en/elasticsearch/reference/2.4/array.html
```When adding a field dynamically, the first value in the array determines the field type.```

Moreover, it is the same as defined in the video mapping:
https://github.com/CERNDocumentServer/cds-videos/blob/cdslabs_qa/cds/modules/records/mappings/records/videos/video/video-v1.0.0.json#L268

I have actually tested it and it works, and before making this change, when importing the `categories`, ES was automatically creating the mapping (because it was not defined) as `string`.

What do you think?

Reply:
👍 thanks for the explanation!